src/class/Context/Context.js

```diff
@@ -97,14 +96,21 @@ export default class Context {
   }
 
   /**
-   *
    * @param {string} method
-   * @param {*} payload
-   * @return {Promise<*>}
+   * @param {any} [payload]
+   * @returns {Promise<any>}
    */
   executeMethod(method, payload) {
+    /**
+     * @param {Function} resolve 
+     * @param {Function} reject 
+     * @returns {Promise<void>}
+     */
     const executor = async (resolve, reject) => {
-      const answer = (result = {}) => {
+      /**
+       * @param {EventResult} result 
+       */
+      const answer = result => {
```

Comment:
result can be undefined too, like the [same](https://github.com/endpass/endpass-connect/pull/153/files#r343002530)

Reply:
@dudiq how result can be undefined, if we destruct it on the next line? There are only 2 cases:
- bug in the code, one little bug in the code (because we cannot destruct `undefined`, it's obvious)
- result cannot be `undefined` (at least, in this case)

For now left unchanged.

Reply:
@dudiq  ?

Reply:
we desctruct, yes, but try to desctruct undefined value, result need to be with default value.

also please check tests.
this pr must not affect to code, and tests shows, that we have affect in some cases

Reply:
@dudiq eslint (if you mean it) fails at parts of code which untouched by me. My changes not affect eslint. tsc fails at @endpass/class module.

Reply:
when i typed about `tests` i mean `yarn test:unit` call) it fails only in `Context.spec.js`

about eslint - tired to run `yarn format` on my wsl bash and have only 6 errors and 13 warnings
about tsc - no errors, all is fine. maybe something broken with symlinks? try to re-init `node_modules` by dropping it and re-call `yarn` in repo.

Reply:
@dudiq eslint fails at another files which wasn't affected by my changes